components/places/src/storage/history.rs

```diff
@@ -233,8 +238,12 @@ pub fn delete_visits_between(db: &PlacesDb, start: Timestamp, end: Timestamp) ->
 }
 
 pub fn delete_place_visit_at_time(db: &PlacesDb, place: &Url, visit: Timestamp) -> Result<()> {
+    delete_str_place_visit_at_time(db, place.as_str(), visit)
+}
+
+pub fn delete_str_place_visit_at_time(db: &PlacesDb, place: &str, visit: Timestamp) -> Result<()> {
```

Comment:
Similarly, rename this to delete_place_visit_at_time_by_href` or something.

Reply:
Done